torch/csrc/jit/passes/quantization.cpp

```diff
@@ -40,39 +40,45 @@ int groups, bool benchmark, bool deterministic, bool cudnn_enabled) -> Tensor"};
   return quantnodeLookup.find(n);
 }
 
-void insertQuantNodeParams(Node* quant, std::tuple<float, int> qparam) {
+void insertQuantNodeParams(Node* quant,
+  std::tuple<std::string, float, int>& qparam) {
+  auto& scale = std::get<1>(qparam);
+  auto& zero_point = std::get<2>(qparam);
   WithInsertPoint ins(quant);
-  Value* scale = quant->owningGraph()->insertConstant(std::get<0>(qparam));
-  Value* zeropoint = quant->owningGraph()->insertConstant(std::get<1>(qparam));
-  quant->addInput(scale);
-  quant->addInput(zeropoint);
+  Value* scale_v = quant->owningGraph()->insertConstant(scale);
+  Value* zeropoint_v = quant->owningGraph()->insertConstant(zero_point);
+  quant->addInput(scale_v);
+  quant->addInput(zeropoint_v);
 }
 
-// Create Quant-Dequant node pair for quantizable Value
-std::pair<Node*, Node*> createQuantDeQuantNodes(Value* v, Node* n) {
-  Node* quant = n->owningGraph()->create(
-      at::Symbol::fromQualString("aten::quantize_linear"));
+// Create Quant Node
```

Comment:
This is not a very helpful comment, it just duplicates what the function name says. At the same time, I don't know what is V and N that we pass into here.